app/helpers/json_schema_validation.rb

```diff
@@ -0,0 +1,49 @@
+require "json-schema"
```

Comment:
I would put this into a rake task (`lib/tasks/json_schema_validation`) so that you can run it from terminal like `rake form_definition:validate` or similar.

Reply:
Probably doesn't really belong in the app since I don't think we'd ever expect to run it at runtime?

Reply:
Getting frustrating error where run from rake it won't load the json-schema library

Reply:
What happens if you require it in the rakefile?

Reply:
Fixed thanks to Dushan. was doing gem install rather than adding to the gemfile and will this ran when called directly. didn't find it from rake.